pkg/config/testdata/piped/piped-config.yaml

```diff
@@ -137,7 +137,7 @@ spec:
     #   encryptServiceAccountFile: /etc/piped-secret/encrypt-service-account.json
 
   imageWatcher:
-    repos:
```

Comment:
One vote from me to keep it as `repos`. It may contain not only the `filter` thing but also other fields (e.g. `interval`).

Reply:
still do not have a good enough ideal for the name but the previous `repos` is lgtm. cc @nakabonne

Reply:
From now my thought has been varied. Let's keep as is.

Reply:
>still do not have a good enough ideal for the name

Yup, I keep thinking the best name :D